source/dub/dub.d

```diff
@@ -557,6 +557,8 @@ class Dub {
 			return;
 		}
 
+		bool dependendPackageFetched;
```

Comment:
It's __dependent__, but I think sth. like `hasDependenciesFetched` or `hasDependentPackagesFetched` is probably even better?